keps/sig-apps/4017-pod-index-label/README.md

```diff
@@ -333,6 +333,8 @@ when drafting this test plan.
 [X] I/we understand the owners of the involved components may require updates to
 existing tests to make this code solid enough prior to committing the changes necessary
 to implement this enhancement.
+[X] unit tests: https://github.com/kubernetes/kubernetes/blob/585971431b4c8ea290e8c754d5806b51630a1938/pkg/controller/statefulset/stateful_set_control_test.go#L246-L250
+[X] integration and/or e2e: TODO
```

Comment:
Have a look at the [template](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#test-plan) and update it accordingly.